packages/lesswrong/components/search/SequencesSearchHit.tsx

```diff
@@ -22,28 +39,38 @@ const styles = (theme: ThemeType): JssStyles => ({
       marginRight: 5,
     }
   },
+  snippet: {
+
+  }
 });
 
 const SequencesSearchHit = ({hit, clickAction, classes}: {
-  hit: Hit<AlgoliaSequence>,
+  hit: Hit<any>,
```

Comment:
Is `hit` supposed to have the type `AlgoliaSequence`?

Reply:
The other search-hit components don't have types atm. When I try adding the types, it throws errors, I think because the interfaces don't have snippets or highlights. For now I was just copying the typing of the other search-hit components but I could dig into setting it up more properly if that seems important.

Reply:
(It looks like each search hit component has a line like `const post = (hit as AlgoliaPost);` where it's doing some type checking. I don't currently understand the deal with that, flagging it for reference)

Reply:
Ah ok, yeah prob not worth spending more time on it right now.

Reply:
@Raemon Maybe get Robert to take a look at the types. I believe this is something of an area he's specialized in.